aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -121,6 +121,7 @@ pub enum FeatureFlag {
     DefaultToConcurrentFungibleBalance,
     LimitVMTypeSize,
     AbortIfMultisigPayloadMismatch,
+    SimulationEnhancement,
```

Comment:
This is a bit vague, what if we enhance simulation again? Can we use a more descriptive name?

Also the AIP includes multiple changes, should they possibly be feature gated separately? I'm less opinionated on this one.

Reply:
Yes, this has multiple changes. We could rename it to `SimulationFlexibilityConsistencyEnhancement` to make it a bit more descriptive.